styles.css

```diff
@@ -286,6 +323,28 @@ nav a:hover {
         border-radius: 25px;
         margin: 10px;
     }
+    .text-overlay {
+        height: 180px;
+        width: 100%;
+        transition: .5s ease;
+        opacity: 0;
+        position: absolute;
+        top: 50%;
+        left: 50%;
+        transform: translate(-50%, -50%);
```

Comment:
this is a good fix, this is more advanced than i would expect at this point, well done

Reply:
thanks :) It took me about 3 hours